standards/alerting.md

```diff
@@ -1,3 +1,45 @@
-# Alerting & Notifications
+# Monitoring & Alarms
 
-This is a placeholder for linking until the standard has been drafted.
\ No newline at end of file
+All components MUST have an alarm set up.
+
+Components MAY have multiple alarms.
+
+Owners SHOULD identify and monitor other key success metrics.
+
+## Alarm Triggers
+
+### Log Messages
+
+A service MUST create a metric filter for [log messages](logging.md) of severity `ERROR` and greater. 
+
+The metric filter MUST be named and namespaced consistently:
+
+- The namespace MUST be `Log Message`
+- The name MUST be in upper camel case and MUST be prefixed with the component name. For example: `BibServiceLogError`.
+
+The RECOMMENDED search for a metric filter is `{ $.levelCode <= 3 }`.
+
+The metric created from the metric filter MUST be used to trigger an alarm. 
+Other Alarms
+
+Additional alarms MAY be set up for other metrics. For example: 
+
+- HTTP errors (5xx, 4xx status codes)
+- Lambda errors
+- Kinesis errors
+
+### Alarm Configuration
+
```

Comment:
Is there a resource you can link to on Alarm Configuration with AWS (that is if developers are using AWS)?